src/mongoose_client_api/mongoose_client_api.erl

```diff
@@ -126,3 +127,17 @@ check_password(User, Password) ->
 is_noauth_http_method(<<"OPTIONS">>) -> true;
 is_noauth_http_method(_) -> false.
 
+%% -------------------------------------------------------------------
+%% @doc
+%% Decode JSON binary into map
+%% @end
+%% -------------------------------------------------------------------
+-spec json_to_map(JsonBin :: binary()) -> {ok, Map :: maps:map()} | {error, invalid_json}.
+
+json_to_map(JsonBin) ->
+    case catch jiffy:decode(JsonBin, [return_maps]) of
+      Map when is_map(Map) ->
```

Comment:
Looks like the indentation is a bit broken. We usually use 4 spaces to indent the code, here are can see 2. Can you adjust the case clause to have 4 spaces. It's about this line and several following lines.

Reply:
Done. Thanks.